### PR TITLE
Update phrases.xml

### PR DESCRIPTION
**Pull request by GiZig**
Better translations

### CODE REVIEW

File: OsmAnd/res/values-fr/phrases.xml [hunk 1]
```diff
@@ -662,19 +662,19 @@
 	<string name="poi_internet_access_service">Accès Internet : service</string>
 	
 	<string name="poi_railway_buffer_stop">Heurtoir ferroviaire</string>
-	<string name="poi_shop_food">Magasin alimentaire</string>
+	<string name="poi_shop_food">Commerce alimentaire</string>
```
Comment (sbrunner) [inline OsmAnd/res/values-fr/phrases.xml]: ??? not french

File: OsmAnd/res/values-fr/phrases.xml [hunk 2]
```diff
@@ -633,13 +633,13 @@
 	<string name="poi_transportation">Transport</string>
 	<string name="poi_man_made">Infrastructure</string>
 	<string name="poi_education">Éducation</string>
-	<string name="poi_administrative">Territoire</string>
+	<string name="poi_administrative">Administration</string>
 	<string name="poi_healthcare">Santé</string>
-	<string name="poi_office">Service</string>
+	<string name="poi_office">Tertiaire</string>
```
Comment (sbrunner) [inline OsmAnd/res/values-fr/phrases.xml]: Is it relay a good translation ?

Comment (GiZig) [inline OsmAnd/res/values-fr/phrases.xml]: It is the better one I've found.
"Bureau" is a literal translation, but it is really very strange for 
French (I assume you speak French, and so you'll agree).
"Service" is not so bad, but it is used for… "Service" POI !
As every POI in this "office" category belongs to tertiary sector, I 
think it is the best way to tag it.

Comment (sbrunner) [inline OsmAnd/res/values-fr/phrases.xml]: Why not `Bureau` ?

Comment (GiZig) [inline OsmAnd/res/values-fr/phrases.xml]: Don't you think "Bureau" is very strange? I think nobody can understand 
which POI would be in such a category "Bureau": shops where you can buy 
pens? :-)

**Comment by sbrunner** [comment]
Then I mis understand the usage of office do you have more information about that? For me a shops where you can buy pens is a `Papeterie`!

**Comment by GiZig** [comment]
"office"(en) category contains POI such as "Agence de voyages", "Agence 
pour l'emploi", "ONG", "mairie", etc.
I said "bureau"(fr) is unclear. French people would think it contains 
POI such "Papeterie" (which sells pens for office).

**Comment by sbrunner** [comment]
Wow it's a real global word, It's clear that we don't have any word for all those things in french... :-(

**Pull request merged**